Automattic/newspack-newsletters
# fix: require postcss explicitly

### All Submissions:






### Changes proposed in this Pull Request:



Resolves a build issue.

`newspack-scripts` uses PostCSS v8, but an indirect dependency must be setting it to a lower version. Requiring v8 explicitly resolves a build issue.

### How to test the changes in this Pull Request:

1. On `master`, try to build (`npm run build`)
2. Observe it logging warnings about PostCSS usage
3. Switch to this branch, build again, observe building without issues

### Other information:
